### DISCUSSION TITLE
Query a date based on another database field

### DISCUSSION THREAD

**Discussion by petrbelecky** [prisma/prisma#8599] [Q&A]
Hi, I have questions and I wasn't able to find an answer anywhere. I have a table with tasks. Every task has dateTime it has been processed and an interval in minutes in which it should be processed. 

Is there a way to query the table to return tasks that should be processed based on the interval? Meaning processed time + interval < current time?

Eg. 
task 1
processed = 19:00
interval = 30
current time = 19:31

task 2
processed = 19:00
interval = 40
current time = 19:31

=> query should return only task 1

Thank you very much for your help!

```prisma
model Task {
  id        Int       @id @unique @default(autoincrement())
  active    Boolean   @default(false)
  name       String
  interval  Int       @default(30)
  processed DateTime?
  createdAt DateTime  @default(now())
  updatedAt DateTime  @updatedAt
}
```

**Comment by ryands17** [comment]
@PetrBelecky 👋 

Unfortunately Prisma doesn't support this out of the box so you would need to perform this with a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access). We have a feature request for the same below so it would be great if you could add a 👍 to this so that we know the priority 🙂 

https://github.com/prisma/prisma/issues/5048

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
